kobo/apps/organizations/models.py

```diff
@@ -12,6 +12,16 @@
 class Organization(AbstractOrganization):
     uid = KpiUidField(uid_prefix='org')
 
+    @property
+    def email(self):
+        """
+        This exists to make dj-stripe happy
+        """
+        return self.owner.organization_user.user.emailaddress_set.get(
```

Comment:
User.email is the canonical email address to read. I know it's not normalized and not clear this should be used. Can you think of a good place to document this? I'm not sure because there is no code we wrote that defines this. It's the upstream Django auth.User and allauth.account.EmailAddress. If you can't think of a better place, maybe a code comment in settings/base.py near the other ACCOUNT_ settings.